physics/equipotential_body.hpp

```diff
@@ -198,11 +195,16 @@ auto Equipotential<InertialFrame, Frame>::ComputeLines(
     SpecificEnergy const& energy) const -> std::vector<State> {
   using WellIterator = typename std::vector<Well>::const_iterator;
   LOG(ERROR) << "V=" << energy;
+
+  // The set of wells that are not yet delineated from a peak by equipotentials
+  // already computed, as well as whether the well is delineated from the “well
```

Comment:
`as well as` -> `together with`, `also`, `plus`, `additionally`, etc.  There are two many wells in this comment. ䷯.